xml/admin_ceph_upgrade.xml

```diff
@@ -6,1852 +6,633 @@
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     In case you previously upgraded from version 4, verify that the upgrade to
-     version 5 was completed successfully:
-    </para>
-    <itemizedlist>
      <listitem>
       <para>
-       Check for the existence of the file
+       The cluster upgrade may take a long time&mdash;approximately the time it
+       takes to upgrade one machine multiplied by the number of cluster nodes.
       </para>
-<screen>/srv/salt/ceph/configuration/files/ceph.conf.import</screen>
+     </listitem>
+     <listitem>
       <para>
-       It is created by the engulf process during the upgrade from SES 4 to 5.
-       Also, the <option>configuration_init: default-import</option> option is
-       set in the file
+       You have to upgrade the &smaster; first, and replace &deepsea; with
+       <literal>ceph-salt</literal> and &cephadm; after.  You will be unable to
+       start using the &cephadm; orchestrator module until at least all &mgr;'s
+       are upgraded.
       </para>
-<screen>/srv/pillar/ceph/proposals/config/stack/default/ceph/cluster.yml</screen>
+     </listitem>
+     <listitem>
       <para>
-       If <option>configuration_init</option> is still set to
-       <option>default-import</option>, the cluster is using
-       <filename>ceph.conf.import</filename> as its configuration file and not
-       &deepsea;'s default <filename>ceph.conf</filename> which is compiled
-       from files in
+       The upgrade from using Nautilus RPMs to Octopus Containers needs to happen all
+       in one step. This means upgrade an entire node at a time, not one daemon
+       at a time.
       </para>
-<screen>/srv/salt/ceph/configuration/files/ceph.conf.d/</screen>
+     </listitem>
+     <listitem>
       <para>
-       Therefore you need to inspect <filename>ceph.conf.import</filename> for
-       any custom configuration, and possibly move the configuration to one of
-       the files in
+        The OSD migration from FileStore to Bluestore <emphasis>needs</emphasis>
+        to happen before the upgrade as it is unsupported in &productname;
+        &productnumber;. See <xref linkend="filestore2bluestore"/> for
+        more information.
       </para>
-<screen>/srv/salt/ceph/configuration/files/ceph.conf.d/</screen>
+    </listitem>
+    <listitem>
       <para>
-       Then remove the <option>configuration_init: default-import</option> line
-       from
+        If you are running an older cluster that still uses <literal>ceph-disk</literal>
+        OSDs, you <emphasis>need</emphasis> to switch to <literal>ceph-volume</literal> before
+        the upgrade.
       </para>
-<screen>/srv/pillar/ceph/proposals/config/stack/default/ceph/cluster.yml</screen>
-      <warning>
-       <title>Default &deepsea; Configuration</title>
+    </listitem>
+    </itemizedlist>
+  </sect2>
+
+   <sect2 xml:id="upgrade-backup-config-data">
```

Comment:
maybe referencing https://documentation.suse.com/ses/6/single-html/ses-admin/#cha-deployment-backup if it covers all needed info; if it does not, we should enhance the old version and republish it